framework/src/media/voice/HardwareEndPointDetector.cpp

```diff
@@ -141,5 +146,47 @@ bool HardwareEndPointDetector::startEndPointDetect(int timeout)
 	return ret;
 }
 
+bool HardwareEndPointDetector::detectEndPoint(short *sample, int numSample)
+{
+	audio_manager_result_t result;
+	uint16_t msgId;
+
+	result = get_device_process_handler_message(mSdCard, mSdDevice, &msgId);
+	if (result == AUDIO_MANAGER_SUCCESS && msgId == AUDIO_DEVICE_SPEECH_DETECT_EPD) {
+		int sval;
+		medvdbg("#### EPD DETECTED!! ####\n");
+		if (sem_getvalue(&mSem, &sval) == OK && sval < 0) {
```

Comment:
Please clean bracket, one bracket per one operation
```
if ((A == B) && (C < D)) {
}

Reply:
Fixed it